Cargo.toml

```diff
@@ -5,11 +5,9 @@ authors = ["Dan Glastonbury <dglastonbury@mozilla.com>"]
 description = "Cubeb backed for PulseAudio written in Rust"
 
 [features]
-default = ["dynamic-link"]
-static-link = ["pulse-ffi/static-link"]
-dynamic-link = ["pulse-ffi/dynamic-link"]
+pulse-dynamic-link = ["pulse-ffi/dynamic-link"]
```

Comment:
Ok, I'll make that change. It should be pulse-ffi/dynamic-load too, because it uses dlopen/dlsym which is dynamic loading not dynamic linking.